src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc

```diff
@@ -20,15 +20,15 @@ The link:https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins[c
 `publisher`: Name of the publisher.
 `repository`: URL of the source repository.
 `title`: Title (long).
-`type`: `Che Plugin`, `VS Code extension`.
+`type`: `{prod-short} Plugin`, `VS Code extension`.
 `version`: Version (example\: `7.5.1`).
 `spec`: Specifications (see underneath).
 :===
 
 .`spec` attributes
 :===
 `endpoints`: plugin endpoints
-`containers`: sidecar containers for plugin. `Che Plugin` and `VS Code extension` supports only one container.
+`containers`: sidecar containers for plugin. `{prod-short} Plugin` and `VS Code extension` supports only one container.
```

Comment:
```suggestion
`containers`: sidecar containers for plug-in. `{prod-short} plug-in` and `VS Code extension` supports only one container.
```

Reply:
I have to revert `Che Plugin` and `Che editor` types.

Reply:
This is confusing: type for a `{prod-short} plug-in` can be: `Che Plugin`, `Che Editor`, `VS Code extension`. Depending on the context we may or may not use the attribute. All my attempts to automate the substitution with regex failed so far. I need to reassess everything manually each time :/.